singleheader/CMakeLists.txt

```diff
@@ -20,7 +20,10 @@ set_source_files_properties(${SINGLEHEADER_FILES} PROPERTIES GENERATED TRUE)
 
 find_program(BASH bash)
 
-if (BASH)
+# Under Windows, exectuting a bash script works, except that you cannot generally
+# do bash C:/path to my script. You need a  "mounted" path: /mnt/c/path
+
+if (BASH AND (NOT WIN32))
```

Comment:
Ugh. We're going to just have to get rid of all of our bash scripts at some point.

Reply:
Well, if we could convert it to a mounted path, it would work. There may already have a really simple way to do this. 

With WSL becoming standard... I bet that people know how to do this.